plugin_store/api/__init__.py

```diff
@@ -65,15 +66,28 @@ async def index():
     return INDEX_PAGE
 
 
+# TODO: move these enums somewhere to deduplicate them
+class SortOption(StrEnum):
+    desc = auto()
+    asc = auto()
+    NONE = auto()
+
+class SortType(StrEnum):
+    name = auto()
+    date = auto()
+    NONE = auto()
```

Comment:
can we use `None` do indicate no sorting instead? Then there would be no field for that in this enum, and sort param would be defined as `Optional[SortOption]` instead.